dependencies.yaml

```
@@ -304,7 +304,7 @@ dependencies:
           - nbsphinx
           - numba>=0.59.1,<0.61.0a0
           - numpydoc
-          - sphinx
+          - sphinx<=8.2.0
```

8.2.0 doesn't work, so this should be an exclusive upper bound.
```suggestion
          - sphinx<8.2.0
```